admin/javascript/src/state/campaign/reducer.js

```diff
@@ -15,6 +15,9 @@ function campaignReducer(state = initialState, action) {
         view: action.payload.view,
       }));
 
+    case ACTION_TYPES.PUBLISH_CAMPAIGN_SUCCESS:
```

Comment:
Why? You're still viewing that campaign after publishing, right? On a related note: What's best practice for cleaning up Redux UI state after a view is "closed"? I guess as long as any new instance of this view derives its new state correctly (e.g. from routing), it doesn't matter too much.